frontend/layouts/default/Footer.tsx

```diff
@@ -36,155 +23,41 @@ import {
    FooterLink,
    FooterDivider,
    EventTracker,
+   NavigationSection,
 } from '@ifixit/footer';
+import { CountryCode } from '@lib/shopify-storefront-sdk';
 
 interface FooterProps {
    stores: StoreListItem[];
-   menu1: Store['footer']['menu1'];
-   menu2: Store['footer']['menu2'];
    partners: Store['footer']['partners'];
    bottomMenu: Store['footer']['bottomMenu'];
    socialMediaAccounts: Store['socialMediaAccounts'];
+   menu1: Store['footer']['menu1'];
+   menu2: Store['footer']['menu2'];
+   menu3: Store['footer']['menu3'];
```

Comment:
Can we make these more generic (e.g. make a `NavigationList` type, replace menu1-3 with a `menus: NavigationList[]` prop)? If it's difficult to change we can save it for later.

Reply:
I don't think we can change this prop type because each of the strapi menu types here are different. I don't think there's a way to generalize these three types together.